cli/command/container/completion_test.go

```diff
@@ -21,6 +25,48 @@ func TestCompleteLinuxCapabilityNames(t *testing.T) {
 	}
 }
 
+func TestCompletePid(t *testing.T) {
```

Comment:
If you decide to change the name then the test should probably also be renamed; `TestCompletePIDMode`